include/RAJA/policy/cuda/forall.hpp

```diff
@@ -141,14 +149,54 @@ template <size_t BlockSize,
 __launch_bounds__(BlockSize, 1) __global__
```

Comment:
@ajkunen did you say that there are two occurrences of the launch bounds thing? This is the only one I see. 

Do we want to remove this or leave it in? I thought the idea was to prevent folks from being hit with mysterious register pressure/spillage.

Reply:
That's the only one.  I'm not sure what to do with it.  I would leave it in for the time being.